
[38mic| model_gpus: [3]
[38mic| next(model.parameters()).is_cuda: True
[38mic| start_epoch: 0, final_checkpoint: None
  0%|                                                                                                                                                                   | 0/5 [00:00<?, ?it/s, loss=3]
  0%|                                                                                                                                             | 9/33932 [00:02<2:24:18,  3.92it/s, model_number=3]
 20%|â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ                                                                                                                            | 1/5 [00:04<00:19,  4.96s/it, loss=3]
  0%|                                                                                                                                             | 9/33932 [00:02<2:06:15,  4.48it/s, model_number=3]
 20%|â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ                                                                                                                            | 1/5 [00:09<00:36,  9.00s/it, loss=3]
Traceback (most recent call last):
  File "train_ddp.py", line 414, in <module>
    main()
  File "train_ddp.py", line 411, in main
    cleanup()
  File "train_ddp.py", line 38, in cleanup
    dist.destroy_process_group()
  File "/home2/aditya_hari/miniconda3/envs/multisent/lib/python3.7/site-packages/torch/distributed/distributed_c10d.py", line 997, in destroy_process_group
    assert pg is not None
AssertionError